crates/fluvio-cli/src/client/partition/list.rs

```diff
@@ -30,6 +33,14 @@ impl ListPartitionOpt {
         let admin = fluvio.admin().await;
 
         let partitions = admin.all::<PartitionSpec>().await?;
+        let partitions = if !self.show_hidden {
```

Comment:
Issue with this approach is that it add more complexity to client and make less flexible to add more policy in the future.  This should be pushed to SC.   There is already `summary` flag in the `ListRequest`.

Reply:
Moved filtration to SC, and renamed to `system`